titus-server-master/src/main/java/com/netflix/titus/master/eviction/service/quota/job/UnhealthyTasksLimitTracker.java

```diff
@@ -87,23 +87,8 @@ public EvictionQuota getQuota(Reference reference) {
             return Pair.of(0, "job not found");
         }
 
-        // Check first how many started tasks we have
-        int started = 0;
-        for (Task task : tasks) {
-            if (task.getStatus().getState() == TaskState.Started) {
-                started++;
-            }
-        }
-        if (started < minimumHealthyCount) {
-            return Pair.of(
-                    0,
-                    String.format("too few containers started: desired=%s, started=%s", JobFunctions.getJobDesiredSize(job), started)
-            );
-        }
-
-        // We have enough tasks started. Check now how many are healthy.
         int healthy = 0;
-        Map<String, String> unhealthyTasks = new HashMap<>();
+        Map<String, String> notStartedAndHealthyTasks = new HashMap<>();
```

Comment:
is this not started and unhealthy tasks or not started and healthy tasks?

Reply:
The meaning is not (started and healthy). I agree the naming is confusing. I will make it more explicit.